static/js/map.js

```diff
@@ -3638,6 +3668,74 @@ function updateMap() {
     })
 }
 
+const getAllParks = function () {
+    if (!showConfig.parks) {
+        return
+    }
+
+    $.getJSON('static/data/parks-ex-raids.json').done(function (response) {
+        if (!response || !('parks' in response)) {
+            return
+        }
+
+        mapData.exEligibleParks = response.parks.map(parkPoints => parkPoints.map(point => L.latLng(point[0], point[1])))
+
+        if (Store.get('showExEligibleParks')) {
+            updateParks()
+        }
+    }).fail(function () {
+        console.log("Couldn't load ex eligible parks JSON.")
+    })
+
+    $.getJSON('static/data/parks-nests.json').done(function (response) {
+        if (!response || !('parks' in response)) {
+            return
+        }
+
+        mapData.nestsParks = response.parks.map(parkPoints => parkPoints.map(point => L.latLng(point[0], point[1])))
+
+        if (Store.get('showNestsParks')) {
+            updateParks()
+        }
+    }).fail(function () {
+        console.log("Couldn't load ex eligible parks JSON.")
+    })
+}
+
+const updateParks = function () {
+    if (!showConfig.parks || (!Store.get('showExEligibleParks') && !Store.get('showNestsParks'))) {
+        return
+    }
+
+    if (Store.get('showExEligibleParks')) {
+        const inBoundParks = mapData.exEligibleParks.filter(parkPoints => {
+            return parkPoints.some(point => {
+                return map.getBounds().contains(point)
+            })
+        })
+
+        exEligibleParksLayerGroup.clearLayers()
+
+        inBoundParks.forEach(function (park) {
+            L.polygon(park, {'color': 'limegreen'}).addTo(exEligibleParksLayerGroup)
```

Comment:
Please add the `interactive` option. This disables mouse events, we don't want those for parks. https://leafletjs.com/reference-1.5.0.html#interactive-layer